docker/assets/jupyter-labhub

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+# Wrapper around jupyter-labhub to enable restarting jupyter-labhub without
+# stopping docker image.
+#
+# This is needed to debug installation of server extensions without re-building
+# entire docker images.
+
+set -o pipefail
+
+APP_LOG="/tmp/labhub.log"
```

Comment:
We shouldn't be diverting logs to a file, I don't think. Better to do it the Docker way and keep them flowing to stdout, and reading them via Kubernetes. That means that logs flow into the main systems still.

Actually, are you just logging the "exited" and "failure" ?

Reply:
And how will I see them in the current deployment?

When debugging broken serverextensions I'm currently completely blind.

Reply:
by the way logs still flow to stdout, that's why I'm using `tee` I'm just trying to make them visible to the developer where developer is.

Reply:
Ah, cool. I didn't know tee did that.

Yeah, I know you want to see the logs without going to Kubernetes. This seems like a neat solution. Only issue is if it logs a lot and the disk fills up for users. But it's in tmp so it is pretty short-term.